### PR TITLE
Release of version 9.0.0

### PR DESCRIPTION
**Pull request by pesektomas**
| Q             | A
| ------------- | ---
|Description, reason for the PR| rc-v9-0-0
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,197 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v9.0.0](https://github.com/shopsys/shopsys/compare/v7.3.3...v9.0.0) - 2020-05-18
```
Comment (TomasLudvik) [inline CHANGELOG.md:33]: Do not forget to update date before merging

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,197 @@ The changelog is generated during the release process using [ChangelogLinker](ht
...
+- [#1622] easier elasticsearch extensibility
+- [#1606] Redis upgrade
+- [#1663] automatically generated entity extension map, Thanks to [@pk16011990]
+- [#1647] upgrade-instructions-for-symfony-flex.md - typo
+- [#1645] export js translations
+- [#1542] switch domain default currencies
+- [#1635] multiple delivery addresses
+- [#1478] bad constraint message for minimum password length, Thanks to [@sspooky13]
+- [#1681] renamed js files
+- [#1758] updated [@shopsys]/framework npm dependencies
```
Comment (TomasLudvik) [inline CHANGELOG.md:184]:
```suggestion
- [#1758] updated shopsys/framework npm dependencies
```

File: CHANGELOG.md (hunk 1 above)
Comment (pesektomas) [reply CHANGELOG.md:33]: i will try it :smiley:

File: CHANGELOG.md (hunk 2 above)
Comment (pesektomas) [reply CHANGELOG.md:184]: :+1:

File: docs/contributing/backward-compatibility-promise.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Once created, a git tag marking a release will never be removed or edited.
     Pre-release version format is `MAJOR.MINOR.PATCH-<alpha|beta|rc><n>`, eg. `7.0.0-beta5`.
 
 ### Current Release Plan
-Currently, `7.3` and `8.1` versions are supported, i.e. we are releasing patch versions for them on a regular basis.
+Currently, `7.3` and `9.0` versions are supported, i.e. we are releasing patch versions for them on a regular basis.
 
 To be able to develop and improve Shopsys Framework we plan to release `MAJOR` versions almost quarterly, aiming to release a new `MAJOR` every 3-4 months.
```
Comment (sslt) [inline docs/contributing/backward-compatibility-promise.md:28]: We want to amend the release cadence, currently we plan to release  major version every 12-15 months

Comment (pesektomas) [reply docs/contributing/backward-compatibility-promise.md]: e05c64d5e31b74e1808c620dc55b417690b94037

**Pull request merged**